### PR TITLE
Fix UI: Home and general section

### PR DESCRIPTION
**Pull request by austin-pham-goldenowl**
https://trello.com/c/iNMBOmUv/256-home-page

### CODE REVIEW

File: src/components/Explore/index.sass [hunk 1]
```diff
@@ -21,7 +21,7 @@
     font-family: ProximaNovaBold
     line-height: 1.23
     padding: 0.9vw 0 4.2vw 0
-    width: calc(57.4% + 120px);
+    width: calc(57.4% + 135px);
```
Comment (hansanwok) [inline src/components/Explore/index.sass]: reduntdant ;

File: src/components/MainHeader/index.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ class MainHeader extends Component {
             <li className="main-header__flex-item">
               <Link to="/portfolio">Portfolio</Link>
             </li>
-            <li className="main-header__flex-item font-bold" style={{ display: 'none' }}>
+            <li className="d-none main-header__flex-item font-bold" style={{ display: 'none' }}>
```
Comment (hansanwok) [inline src/components/MainHeader/index.js]: both d-none and style???

File: src/components/SubHeader/index.js [hunk 3]
```diff
@@ -66,7 +67,46 @@ class SubHeader extends Component {
...
+        {/* */}
+        <Dropdown>
+          {/* <button
+            className="dropdown-toggle btn btn-link services__arrow-wrapper sub-header__button no-hover-text-decoration d-flex no-bootstrap-arrow"
+            type="button"
+            dataToggle="dropdown"
+            id="services-dropdown"
+            ariaHaspopup="true"
+            ariaExpanded="false"
+          > */}
```
Comment (hansanwok) [inline src/components/SubHeader/index.js]: many unused comment

File: src/components/SubHeader/service.sass [hunk 4]
```diff
@@ -3,17 +3,9 @@
...
-    padding-right: 30px
-    padding-top: 20px
-    padding-bottom: 20px
-    background-color: $white
     width: 434px
-    border-radius: 10px
-
-    z-index: 50
+    border-radius: 10px;
+    box-shadow: 10px 10px 20px -15px rgba(160, 160, 160, 0.5);
```
Comment (hansanwok) [inline src/components/SubHeader/service.sass]: ;

File: src/pages/Home/components/Introduction/index.js [hunk 5]
```diff
@@ -4,18 +4,20 @@ import study from '../../../../assets/images/study.png'
...
-      <p className="introduction__text">
-      The role of Golden Owl Asia starts where Enterprise meets technology. We provide IT solutions that allow our Clients to achieve specific goals and measurable benefits.
-        <br />
-        <br />
-      Working with us would mean working with a virtual team in a different time zone so that effective communication remains the number one issue.  Quality of operations and accelerate speed to the market are improved with scalable and flexible services.
-      </p>
+  <section className="introduction">
+    <div className="row">
+      <div className="col-md-5 introduction__left-side">
+        <img src={study} alt={study} loading="lazy" />
```
Comment (hansanwok) [inline src/pages/Home/components/Introduction/index.js]: src and alt use same value?

File: src/components/MainHeader/index.js (hunk 2 above)
Comment (austin-pham-goldenowl) [reply src/components/MainHeader/index.js]: that button is hidden for now

**Pull request merged**